cartoframes/vis/map.py

```diff
@@ -30,74 +28,78 @@ class Map(object):
           properties, or a list of floats in the following order: [west,
           south, east, north]. If not provided the bounds will be automatically
           calculated to fit all features.
+        size (tuple of int): a (width, height) pair for the size of the map.
+          Default is (1024, 632)
 
     Example:
 
         .. code::
 
-            from cartoframes import carto_vl as vl
-            from cartoframes import CartoContext
+            from cartoframes import Context, set_default_context
+            from cartoframes.vis import Map, Layer
 
-            context = CartoContext(
+            context = Context(
```

Comment:
Is the plan to change `CartoContext` -> `Context`? I don't see that change reflected in context.py?

Reply:
I just use `CartoContext` as `Context`. I don't know if we can rename `CartoContext` because this breaks the previous API.

Reply:
Nop IMO, at the end in v1.0.0 we will have at least CartoContext (or RemoteContext) and LocalContext